### PR TITLE
update developers details for the pom

### PR DESCRIPTION
**Pull request by pjfanning**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -18,13 +18,13 @@ lazy val publishSettings = Seq(
...
   sonatypeCredentialHost := apacheBaseRepo,
   sonatypeProfileName := "org.apache.pekko",
-  startYear := Some(2022),
+  startYear := Some(2023),
   developers := List(
     Developer(
-      "pekko-contributors",
-      "Apache Pekko Contributors",
+      "pekko-sbt-paradox-contributors",
+      "Apache Pekko Sbt Paradox Contributors",
```
Comment (jrudolph) [inline build.sbt:25]: I don't mind but do we really have to make this distinction everywhere?

**Review by jrudolph — APPROVED** [review]
lgtm

File: build.sbt (hunk 1 above)
Comment (pjfanning) [reply build.sbt:25]: * it's what we've been doing in other repos
* an example is that a contributor could contribute to this repo but not to the main pekko one - so the link would not acknowledge their work

Comment (pjfanning) [reply build.sbt:25]: yes - only the link technically needs to be different but the point is that we sort of have this norm in the other repos too (being specific for all the developer params)

**Pull request merged**